concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -135,65 +150,201 @@ def main():
     api_request(
         cluster_url,
         'receive',
         REQUESTOR_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
-        headers = {
+        headers={
             'Content-Type': 'application/octet-stream',
             'concent-client-public-key': b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
-        }
+        },
+        expected_status  = 200,
+        expected_message = ForceGetTaskResultUpload
     )
 
-    # Case 2 - test for non existing file
+
+@count_fails
+def case_4c_concent_reports_a_failure_to_get_task_results_if_file_has_bad_hash(cluster_url, current_time, task_id):
+    """
+    Requestor -> Concent:    ForceGetTaskResult
+    Concent   -> Requestor:  ForceGetTaskResultAck
+    Concent   -> Provider:   ForceGetTaskResult + FileTransferToken
+    Provider  -> Concent:    Upload is done with bad file
+    Concent   -> Requestor:  ForceGetTaskResultFailed (Any of the files had a hash or size that did not match ReportComputedTask)
+    """
+    file_check_sum, file_content, file_size, headers = prepare_file_for_transfer('0', task_id, TOKEN_EXPIRATION_TIME)
+
     api_request(
         cluster_url,
         'send',
         REQUESTOR_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
-        get_force_get_task_result(
-            task_id + 'non_existing_file',
-            current_time,
-            size    = 1024,
-            checksum = '098f6bcd4621d373cade4e832627b4f6'
-        ),
+        get_force_get_task_result(task_id, current_time, size = file_size, checksum = file_check_sum),
         headers = {
-            'Content-Type':                     'application/octet-stream',
-            'concent-client-public-key':        b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
-            'concent-other-party-public-key':   b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
-        }
+            'Content-Type': 'application/octet-stream',
+            'concent-client-public-key': b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
+            'concent-other-party-public-key': b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
+        },
+        expected_status  = 200,
+        expected_message = AckForceGetTaskResult
     )
-    time.sleep(10)
-
+    print(f"Waiting {WAIT_TIME_1} seconds...")
+    time.sleep(WAIT_TIME_1)  # TaskToCompute.deadline + FORCE_ACCEPTANCE_TIME < current time <= TaskToCompute.deadline + FORCE_ACCEPTANCE_TIME + CONCENT_MESSAGING_TIME
     api_request(
         cluster_url,
         'receive',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         headers = {
-            'Content-Type':                 'application/octet-stream',
-            'concent-client-public-key':    b64encode(PROVIDER_PUBLIC_KEY).decode('ascii')
-        }
+            'Content-Type': 'application/octet-stream',
```

Comment:
Please align. Actually there is a more of those below, please fix them.

Reply:
done

Reply:
done